LICENSE

```diff
@@ -1,6 +1,6 @@
-MIT License
+UNLICENSED License
 
-Copyright (c) 2018 FRENDS
+Copyright (c) 2020 Community HiQ
```

Comment:
Please update the license to 2022 HiQ Finland

Reply:
Fixed